dbms/src/Server/Server.cpp

```diff
@@ -107,6 +108,7 @@
 #include <fiu.h>
 #endif
 
+std::atomic<UInt64> tranquil_time_rss;
```

Comment:
TODO could be moved into KVStore to avoid this atomic. Not necessarily in this PR.